docs/App.Navigation.md

```diff
@@ -416,7 +639,7 @@ By this point you have:
 * Learned how to [style the navigator](https://wix.github.io/react-native-navigation/docs/style-options) using the Options object and how to dynamically merge options.
 
 You can view the full project in [this repository](
-https://github.com/wix-playground/wix-mobile-crash-course).
+https://github.com/wix-playground/wix-mobile-crash-course). It's originally written with class components. Functional components version can be found in [this branch](https://github.com/wix-incubator/react-native-crash-course/tree/functional-components-hooks)
```

Comment:
do we plan to merge this or left for all the time?

Reply:
let's merge and update docs accordingly